.github/format-check.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then 
     git fetch origin $GITHUB_BASE_REF:refs/remotes/origin/$GITHUB_BASE_REF
-    for f in $(git diff --diff-filter=AM --name-only origin/$GITHUB_BASE_REF 'vita3k/**.cpp' 'vita3k/**.h' 'tools/**.cpp' 'tools/**.h'); do
+    for f in $(git diff --diff-filter=AM --name-only origin/$GITHUB_BASE_REF 'vita3k/**.cpp' 'vita3k/**.h' 'vita3k/**.hpp' 'tools/**.cpp' 'tools/**.h' 'tools/**.hpp'); do
```

Comment:
I think for consistency, it is better to rename the only .hpp in the project to .h (there is no reason to have both .hpp and .h files)

Reply:
Done